tests/Unit/Bindings/Browser/ObjectServiceTest.php

```diff
@@ -291,7 +294,16 @@ public function createDocumentDataProvider()
             'Create document without stream' => array(
                 Url::createFromUrl(
                     self::BROWSER_URL_TEST
-                    . '?propertyId[0]=cmis:name&propertyValue[0]=name.jpg&cmisaction=createDocument&succinct=false'
+                ),
+                array(
+                    'cmisaction' => 'createDocument',
+                    'succinct' => 'false',
+                    'propertyId' => array(
```

Comment:
The key was `propertyId[0]' in the GET parameter list and now it is `propertyId`. That's a problem, isn't it?